app/presenters/region_presenter.rb

```diff
@@ -58,14 +58,14 @@ def total_polygons_percentage
   def geometry_ratio
     polygons_count = 0
     points_count = 0
-    total = 0
-
+    total = polygons_count + points_count
```

Comment:
Isn't that going to stay `0` anyway? at the moment of assigning the value to `total` the other 2 variables are 0